app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,13 @@
+class SessionsController < ApplicationController
+  def create
+    access_token = request.env["omniauth.auth"]
+    user = User.from_omniauth(access_token)
+    user.token = access_token.credentials.token
```

Comment:
@rrabinovitch how come this line has to be in here even though `user.token` is already being set in the `User#from_omniauth()` method?

Reply:
Not sure, let's try commenting it out later and see if it breaks!